L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc

```diff
@@ -104,9 +121,9 @@ void HGCalConcentratorSuperTriggerCellImpl::assignSuperTriggerCellEnergyAndPosit
                              ? double(kTriggerCellsForDivision_)
                              : double(superTCmapping_.getConstituentTriggerCells(stc.getSTCId()).size());
 
-    c.setHwPt(stc.getSumHwPt() / denominator);
-    c.setMipPt(stc.getSumMipPt() / denominator);
-    c.setPt(stc.getSumPt() / denominator);
+      c.setHwPt(compressed_value / denominator );
```

Comment:
It will truncate the float to the int below. The effect will probably be very small, but it might be better to round.